src/textual/widget.py

```diff
@@ -1493,6 +1493,43 @@ def scroll_to(
 
         return scrolled_x or scrolled_y
 
+    def scroll_to(
+        self,
+        x: float | None = None,
+        y: float | None = None,
+        *,
+        animate: bool = True,
+        speed: float | None = None,
+        duration: float | None = None,
+        easing: EasingFunction | str | None = None,
+        force: bool = False,
+    ) -> None:
+        """Scroll to a given (absolute) coordinate, optionally animating.
+
+        Args:
+            x: X coordinate (column) to scroll to, or None for no change. Defaults to None.
+            y: Y coordinate (row) to scroll to, or None for no change. Defaults to None.
+            animate: Animate to new scroll position. Defaults to True.
+            speed: Speed of scroll if animate is True. Or None to use duration.
+            duration: Duration of animation, if animate is True and speed is None.
+            easing: An easing method for the scrolling animation. Defaults to "None",
+                which will result in Textual choosing the default scrolling easing function.
+            force: Force scrolling even when prohibited by overflow styling. Defaults to `False`.
```

Comment:
Please remove all the defaults from the docstring :)

Reply:
I see this was likely a copy & paste from other methods, like `scroll_relative`.
If you want, you can also delete the defaults from there 😆

Reply:
Let's make this a followup "clean widget.py docs" PR in that case (although I do tend to like to see something in the docstring that makes it clear it's optional and what the default may be and why).

Reply:
Right. Explaining what the default does or why it is a specific value makes more sense than just writing "Defaults to x".
I don't like "Defaults to x" because it ages badly: in a couple of iterations it will become a lie and it is easy to forget updating that.

A more complete sentence saying something like "Defaults to `None`, which will result in Textual [...]" is _more_ sensible but it can also be rewritten in a way that doesn't even specify the default, e.g., "An easing method for the scrolling animation to replace the one Textual chooses by default".

What do you think?

Reply:
But given the follow-up issue, I think this can be merged.

Reply:
> What do you think?

I think if we explain that something is optional, but don't say what the default is and means, that makes more work for the reader. So *"An easing method for the scrolling animation to replace the one Textual chooses by default"* means either a) we've got to somehow link to somewhere that says what that default is (and so the reader ends up being taken out of their current context and then has to remember they need to go back rather than disappear down a documentation rabbit hole) or b) the reader has to make a note to go find how and where that default behaviour is documented.

I'm totally on board with the whole thing of not repeating facts of the code in the docs, for fear if it going stale, but I do worry that we end up leaving the docs lacking in useful information.

(and now I've seen I've written a fair bit here perhaps this is a consideration for us to spin out into its own issue)